lib/winston/logger.js

```diff
@@ -118,14 +118,24 @@ Logger.prototype.extend = function (target) {
 //
 Logger.prototype.log = function (level) {
   var self = this,
-      args = Array.prototype.slice.call(arguments, 1);
+      timestamp,
+      args;
+
+  if (new Date(level).toString() !== 'Invalid Date') {
```

Comment:
@JiCiT Could you compare `new Date(level).toString()` to doing `isNaN(Date.parse('asd'))`? I think the second option could be cheaper.